pyls_mypy/plugin.py

```diff
@@ -41,17 +44,45 @@ def parse_line(line, document=None):
 
 @hookimpl
 def pyls_lint(config, document):
+    args = ['--incremental',
+            '--show-column-numbers',
+            '--follow-imports', 'silent']
+
+    config_files = config.plugin_settings('pyls_mypy').get('config_files', [])
+    config_files = (os.path.expandvars(os.path.expanduser(p.strip()))
+                    for p in config_files)
+    config_files = [p for p in config_files if p]
+    while config_files:
+        # This seems a little complicated but the idea is that configuration
+        # files in relative paths that are not separated by absolute path
+        # configuration files, will all have an equal chance of being found
+        # in each directory before the search moves up the project tree.
+        config_file = None
```

Comment:
> `config_file` may be named, for example, `found_config_file`.

Or, `maybe_config_file` probably would be better, since we got an ADT here *(algebraic data type)*.

Reply:
That a fair point. I think `found_config_file` would make more sense. Using conventions from the ML family of languages in Python would probably be confusing for most idiomatic Python programmers. I mean, "Maybe" isn't even universally used among languages that include an ADT that represents "some value or no value".